crates/air-lib/interpreter-data/src/executed_state.rs

```diff
@@ -58,8 +60,8 @@ pub enum CallResult {
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
 #[serde(rename_all = "snake_case")]
 pub enum Value {
```

Comment:
Shouldn't it be renamed to `ValueRef` or something like that?

Reply:
Agree, `ValueRef` looks more suitable here.